docs/_customer-api/_example-key-re-generate-StreamResponseDTO.md

```diff
@@ -0,0 +1,87 @@
+```json
+{
+  "streamId": "626fa59fe4b018f4ecdc092c",
+  "customerId": "6267d661e4b0420e3acced5e",
+  "name": "uran",
+  "apiKey": "1651484065733_118447bc58266f92d412ced8ba1bab78b6cca802ab6c5fd93713751f75f80feab9883e1bbb57bf04ed475fc3d04400ae7be26aaf88e146762a02731d66d5e7dc",
+  "encoderIds": [],
+  "currentMonthStreamTime": 0,
+  "previousMonthStreamTime": 0,
+  "qualityPreset": "CUSTOM",
+  "streamKey": "62e297fce4b08b9e2c49580b",
```

Comment:
We should not post streamKey and tokenAuthKey on our documentation site.
Can you replace this with a fake value?